Tests/SwiftLintFrameworkTests/DocumentationTests.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  DocumentationTests.swift
+//  SwiftLint
+//
+//  Created by Marcelo Fabri on 08/24/17.
+//  Copyright © 2017 Realm. All rights reserved.
+//
+
+import Foundation
+import SwiftLintFramework
+import XCTest
+
+private let projectRoot = #file.bridge()
+    .deletingLastPathComponent.bridge()
+    .deletingLastPathComponent.bridge()
+    .deletingLastPathComponent
+
+class DocumentationTests: XCTestCase {
+    func testRulesDocumentationIsUpdated() throws {
+        let docsPath = "\(projectRoot)/Rules.md"
+        let existingDocs = try String(contentsOfFile: docsPath)
+        let updatedDocs = masterRuleList.generateDocumentation()
+
+        XCTAssertEqual(existingDocs, updatedDocs)
+
+        let overwrite = false // set this to true to overwrite existing docs with the generated ones
+        if existingDocs != updatedDocs && overwrite {
+            try updatedDocs.data(using: .utf8)?.write(to: URL(fileURLWithPath: docsPath))
```

Comment:
I copied this approach from SourceKitten but I wonder if we shouldn't just always update the docs 🙃

Reply:
yes. the reason it's parameterized in SourceKitten is that its output is non-deterministic.